pkg/driverutil/disk.go

```diff
@@ -50,7 +50,7 @@ func EnsureDisk(ctx context.Context, instDir, diskSize string, diskImageFormat i
 			diffDiskF.Close()
 			return fmt.Errorf("failed to create sparse diff disk %q: %w", diffDisk, err)
 		}
-		return diffDiskF.Close()
+		diffDiskF.Close()
```

Comment:
How did you test this PR?
The `diffDisk` created in L43  will be overwritten by a new disk converted from `baseDisk` in L57, no?